src/App.js

```diff
@@ -56,7 +56,9 @@ export default function App() {
         <Route path="*" element={<NotFound />} />
         <Route
           path="/"
-          element={<Home isAdult={isAdult} handleCheckIsAdult={checkIsAdult} />}
+          element={
+            <Home isAdult={isAdult} onCheckIsAdult={handleCheckIsAdult} />
```

Comment:
Hi Sven, 
naming drives me always a little bit crazy - but i think you have here the props and would name it `handleCheckIsAdult={handleCheckIsAdult}` to keep it clear when you pass on this props